protostar/cheatable_starknet/callable_hint_locals/call_hint_local.py

```diff
@@ -29,19 +32,21 @@ def name(self) -> str:
 
     def _build(
         self,
-    ) -> Callable[[RawAddress, str, Optional[CairoOrPythonData]], CairoOrPythonData]:
+    ) -> Callable[[RawAddress, int, Optional[CairoOrPythonData]], CallResult]:
         return self.call
 
     def call(
         self,
         contract_address: RawAddress,
-        function_name: str,
+        function_selector: int,
```

Comment:
this should be `CairoShortString`

Reply:
[b5cee22](https://github.com/software-mansion/protostar/pull/1726/commits/b5cee226e668cd3986f0ceaa232c696a7543515b)